### ISSUE TITLE
file_picker ^3.0.0 is incompatible with amplify_flutter

### ISSUE THREAD

**Issue by Ajmaljalal** [aws-amplify/amplify-flutter#461]
When runing flutter pub get this error is thrown and the process ends
```
Because no versions of file_picker match >3.0.0 <4.0.0 and file_picker 3.0.0 depends on plugin_platform_interface ^2.0.0, file_picker ^3.0.0 requires plugin_platform_interface ^2.0.0.
And because every version of amplify_flutter depends on plugin_platform_interface ^1.0.1, file_picker ^3.0.0 is incompatible with amplify_flutter.
So, because forus depends on both file_picker ^3.0.0 and amplify_flutter <1.0.0, version solving failed.
pub get failed (1; So, because forus depends on both file_picker ^3.0.0 and amplify_flutter <1.0.0, version solving failed.)
exit code 1
```

1. add file_picker: ^3.0.0 and flutter_ampliyf to pubspec.yml
2. run pub get

**Expected behavior**
It should not conflict with a version of a another package

**Comment by masterlonghorn** [comment]
I experienced this as well. Similar error with firebase packages as well.

**Comment by haverchuck** [comment]
Yes that's correct @masterlonghorn,  this is related to #434.  We have a fix in progress.

**Comment by SalahAdDin** [comment]
It happens with `uuid` too:
```shell
[thesis_cancer] flutter pub get
Running "flutter pub get" in thesis_cancer...                   
Because freezed >=0.14.1+1 depends on build ^2.0.0 which depends on convert ^3.0.0, freezed >=0.14.1+1 requires convert ^3.0.0.

Because uuid 2.2.2 depends on convert ^2.0.0 and no versions of uuid match >2.2.2 <3.0.0, uuid ^2.2.2 requires convert ^2.0.0.

Thus, freezed >=0.14.1+1 is incompatible with uuid ^2.2.2.

And because amplify_api_plugin_interface 0.1.0 depends on uuid ^2.2.2, freezed >=0.14.1+1 is incompatible with amplify_api_plugin_interface 0.1.0.

And because amplify_flutter 0.1.0 depends on amplify_api_plugin_interface 0.1.0 and no versions of amplify_flutter match >0.1.0 <0.2.0, freezed >=0.14.1+1 is incompatible with amplify_flutter ^0.1.0.

So, because thesis_cancer depends on both amplify_flutter ^0.1.0 and freezed ^0.14.1+1, version solving failed.
pub get failed (1; So, because thesis_cancer depends on both amplify_flutter ^0.1.0 and freezed ^0.14.1+1, version solving failed.)
exit code 1
```

**Comment by haverchuck** [comment]
@SalahAdDin @Ajmaljalal @masterlonghorn This should be resolved by the 0.1.1 release.
